src/audio/CMakeLists.txt

```diff
@@ -73,12 +77,14 @@ check_optimization(fma -mfma -DOPS_FMA)
 check_optimization(hifi2ep -mhifi2ep -DOPS_HIFI2EP)
 check_optimization(hifi3 -mhifi3 -DOPS_HIFI3)
 
-set(sof_audio_modules volume src)
-
-# sources for each module
-set(volume_sources volume.c volume_generic.c)
+set(sof_audio_modules src)
 set(src_sources src.c src_generic.c)
 
+if(CONFIG_VOLUME)
```

Comment:
Host/testbench build doesn't have Kconfig atm, so shouldn't rely on CONFIG_* values because they will be undefined always

Reply:
Removed